go.mod

```diff
@@ -7,35 +7,35 @@ require (
 	github.com/google/go-cmp v0.6.0
 	github.com/prometheus/client_golang v1.20.5
 	github.com/spf13/cobra v1.8.1
-	github.com/spf13/pflag v1.0.5
+	github.com/spf13/pflag v1.0.6
 	github.com/stretchr/testify v1.9.0
-	golang.org/x/oauth2 v0.24.0
+	golang.org/x/oauth2 v0.26.0
 	google.golang.org/api v0.199.0
 	gopkg.in/gcfg.v1 v1.2.3 // indirect
 	gopkg.in/warnings.v0 v0.1.2 // indirect
-	k8s.io/api v0.31.2
-	k8s.io/apimachinery v0.31.2
-	k8s.io/apiserver v0.31.2
-	k8s.io/client-go v0.31.2
-	k8s.io/code-generator v0.31.2
-	k8s.io/component-base v0.31.2
-	k8s.io/component-helpers v0.31.2
-	k8s.io/controller-manager v0.31.2
+	k8s.io/api v0.32.1
+	k8s.io/apimachinery v0.32.1
+	k8s.io/apiserver v0.31.5
+	k8s.io/client-go v0.32.1
```

Comment:
```suggestion
	k8s.io/api v0.31.5
	k8s.io/apimachinery v0.31.5
	k8s.io/apiserver v0.31.5
	k8s.io/client-go v0.31.5
```

Reply:
oh no no, I don't know how this happened, some dep must have updated these

all I did was `go get -u k8s.io/...@v0.31.5` on the k8s.io packages. Not sure how it forced 0.32 for some of them

in any case, I updated the PR